### PR TITLE
Add before_create :initialize_board! to game.rb

### PR DESCRIPTION
**Pull request by washburnad**

### CODE REVIEW

File: test/controllers/games_controller_test.rb [hunk 1]
```diff
@@ -28,6 +28,19 @@ class GamesControllerTest < ActionController::TestCase
     assert_equal expected, game.white_player_id
   end
 
+  test "game created with :create action" do
+    player = FactoryGirl.create(:player)
+    sign_in player
+    game = FactoryGirl.create(:game, :white_player_id => player.id )
```
Comment (kenmazaika) [inline test/controllers/games_controller_test.rb]: I hate the ruby 1.9 hash syntax.  Haha.

**Comment by kenmazaika** [comment]
@houndci is a beast

**Comment by washburnad** [comment]
feels like a nagging spouse already. :-) But I guess it keeps us honest.

http://www.google.com/imgres?imgurl=http://2.bp.blogspot.com/-VMefc9Jlnh4/U2PdwCsQdVI/AAAAAAAAAeQ/j5cx7QNn4hQ/s1600/marge.png&imgrefurl=http://hellogiggles.com/double-standards-dating/2&h=375&w=500&tbnid=beWkruL3hk6s8M:&zoom=1&docid=w0bmjX4JIih46M&ei=SRn6VPD0M6PIsQSWqoLoCw&tbm=isch&ved=0CB0QMygBMAE

File: test/controllers/games_controller_test.rb (hunk 1 above)
Comment (johnellison) [inline test/controllers/games_controller_test.rb]: Yeah @kenmazaika I've noticed that. You're a hashrocket lover for sure.

**Comment by johnellison** [comment]
Maybe Hound was a bad idea?

**Comment by washburnad** [comment]
@johnellison, no I think it was a good idea. Better for us to work to conform to best practices, right?

**Pull request merged**